libraries/AP_HAL_PX4/CAN.h

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>
+ *
+ * With modifications for Ardupilot CAN driver
+ * Copyright (C) 2017 Eugene Shamaev
+ */
+
+#pragma once
+
+#include "AP_HAL_PX4.h"
+#include <systemlib/perf_counter.h>
+#include <AP_HAL/CAN.h>
+#include <pthread.h>
+#include <semaphore.h>
+
+#include "bxcan.h"
+#include "AP_HAL/utility/RingBuffer.h"
+
+#define UAVCAN_STM32_NUM_IFACES 2
```

Comment:
Thank you for the name change. This is still in the header though. I think Pixracer only has one CAN interface (or at least that's what PX4 have in their repo) and it's possible that happens with other boards too.

I'm fine with this going in like this now though as it was already wrong (defined in common) before.